probe-rs/src/vendor/mod.rs

```diff
@@ -180,7 +180,7 @@ fn try_detect_arm_chip(
 fn try_detect_riscv_chip(registry: &Registry, probe: &mut Probe) -> Result<Option<Target>, Error> {
     let mut found_target = None;
 
-    probe.select_jtag_tap(0)?;
+    _ = probe.select_jtag_tap(0);
```

Comment:
Should we maybe report the failure in a log? After all this is discovery and it might be useful to get some info :)

Reply:
It's just noise, not an error. Probe drivers (like cmsis-dap, for reasons) may or may not have scanned the chain at this point.